src/stubs/cssVar.js

```diff
@@ -14,16 +14,16 @@
 
 // If you change these, you'll need to restart the dev server for it to take effect.
 var CSS_VARS = {
-  'scrollbar-face-active-color': '#7d7d7d',
-  'scrollbar-face-color': '#c2c2c2',
-  'scrollbar-face-margin': '4px',
-  'scrollbar-face-radius': '6px',
-  'scrollbar-size': '15px',
-  'scrollbar-size-large': '17px',
-  'scrollbar-track-color': '#fff',
-  'border-color': '#d3d3d3',
-  'fbui-white': '#fff',
-  'fbui-desktop-background-light': '#f6f7f8',
+  '--scrollbar-face-active-color': '#7d7d7d',
```

Comment:
The latest version of `postcss-custom-properties` requires you to specify variables with `--` prefixed